### PR TITLE
Calculator UI

### PR DESCRIPTION
**Pull request by Addisu87**
I have created a calculator user interface that displays all buttons and operators.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,27 +1,73 @@
...
+  background-color: var(--screen-color);
+  border: 0;
+  color: rgb(146, 143, 143);
+  padding: 10px;
+  text-align: right;
+  display: flex;
+  justify-content: flex-end;
+  font-size: 2.28rem;
+  height: 76px;
+}
```
Comment (Lameck1) [inline src/index.css:52]: - [ ] This input field is not properly aligned. This is caused by the width properties given to it and also the one given to `.rowBtn`. We will talk about what to do with `.rowBtn` next.

  Let's fist remove the `max-width` and instead give it a `width: 100%`.

File: src/index.css [hunk 2]
```diff
@@ -1,27 +1,73 @@
...
+  margin: 40px;
+  max-width: 480px;
+  padding: 10px;
+  width: 100%;
+}
+
+.rowBtn {
+  width: 400px;
+  display: grid;
+}
```
Comment (Lameck1) [inline src/index.css:31]: - [ ] Next, let's give the `.rowBtn` a width of 100% too. Using fixed widths is what's creating the problem.

File: src/index.css (hunk 1 above)
Comment (Lameck1) [inline src/index.css]: - [ ] This input field is not properly aligned. This is caused by the width properties given to it and also the one is given to `.rowBtn`. We will talk about what to do with `.rowBtn` next.

  Screenshot 2022-06-29 021925


  Let's fist remove the `max-width` and instead give it a `width: 100%`.

File: src/index.css (hunk 2 above)
Comment (Lameck1) [inline src/index.css]: - [ ] Next, let's give the `.rowBtn` a width of 100% too. Using fixed widths is what's creating the problem.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @Addisu87, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**Please, remember to tag me in your question so I can receive the notification.**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by FrederickMih — APPROVED** [review]
Hi @Addisu87 , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**